packages/key-utils/package.json

```diff
@@ -14,8 +14,9 @@
     "@stablelib/ed25519": "^1.0.3",
     "@stablelib/sha256": "^1.0.1",
     "@stablelib/sha512": "^1.0.1",
+    "@trust/keyto": "^1.0.1",
     "@veramo/core": "4.2.0",
-    "@sphereon/ssi-types": "0.28.1-next.60",
+    "@sphereon/ssi-types": "0.29.1-next.46",
```

Comment:
Please do not update individual deps on SDK. That needs to be done for the whole project or not

Reply:
reverted it